src/Nest/Mapping/Types/Specialized/Attachment/AttachmentProperty.cs

```diff
@@ -81,6 +83,7 @@ public ITextProperty TitleField
 		}
 	}
 
+	[Obsolete("Removed in 6.0.")]
 	[DebuggerDisplay("{DebugDisplay}")]
 	public class AttachmentPropertyDescriptor<T>
```

Comment:
`AttachmentPropertyDescriptor<T>` can be removed altogether in 6.x - it's used for the mapper-attachment plugin in 5.x, which is superseded by the ingest-attachment plugin.